release-notes/9.0/supported-os.md

```diff
@@ -42,81 +43,82 @@ Notes:
-[24]: https://www.microsoft.com/windows/
-[25]: https://support.microsoft.com/help/13853/windows-lifecycle-fact-sheet
-[26]: https://www.microsoft.com/windows-server
+[23]: https://learn.microsoft.com/virtualization/windowscontainers/manage-containers/container-base-images
+[24]: https://learn.microsoft.com/windows-server/get-started/windows-server-release-info
+[25]: https://www.microsoft.com/windows/
+[26]: https://support.microsoft.com/help/13853/windows-lifecycle-fact-sheet
+[27]: https://www.microsoft.com/windows-server
 
 ## Linux compatibility
 
 Microsoft-provided [portable Linux builds](../../linux.md) define minimum compatibility primarily via libc version.
 
-Libc            | Version | Architectures         | Source
---------------- | ------- | --------------------- | --------------
-glibc           | 2.23    | Arm64, x64            | Ubuntu 16.04
-glibc           | 2.35    | Arm32                 | Ubuntu 22.04
-musl            | 1.2.2   | Arm32, Arm64, x64     | Alpine 3.13
-
-Note: Microsoft-provided portable Arm32 glibc builds are supported on distro versions with a [Y2038 compatible glibc](https://github.com/dotnet/core/discussions/9285), for example Debian 12, Ubuntu 22.04, and higher versions.
+| Libc          | Version | Architectures         | Source       |
+| ------------- | ------- | --------------------- | ------------ |
+| glibc         | 2.23    | Arm64, x64            | Ubuntu 16.04 |
+| glibc         | 2.35    | Arm32                 | Ubuntu 22.04 |
+| musl          | 1.2.2   | Arm32, Arm64, x64     | Alpine 3.13  |
 
 ## Notes
 
 * The [QEMU](https://www.qemu.org/) emulator is not supported to run .NET apps. QEMU is used, for example, to emulate Arm64 containers on x64, and vice versa.
+* Microsoft-provided portable Arm32 glibc builds are supported on distro versions with a [Y2038 compatible glibc](https://github.com/dotnet/core/discussions/9285), for example Debian 12, Ubuntu 22.04, and higher versions.
 
-## Out of support OS versions
+## Out of support
 
-Support for the following operating system versions has ended.
+The following operating system versions are no longer supported.
 
-OS                      | Version       | Date
------------------------ | ------------- | ----------------------
-iOS                     | 16            | [2024-08-07](https://support.apple.com/HT213407)
-iPadOS                  | 16            | [2024-08-07](https://developer.apple.com/documentation/ios-ipados-release-notes/ipados-16-release-notes)
-openSUSE Leap           | 15.5          | 2024-12-31
+| OS                    | Version       | Date                 |
+| --------------------- | ------------- | -------------------- |
+| iOS                   | 16            | [2024-08-07](https://support.apple.com/HT213407) |
+| iPadOS                | 16            | [9999-12-31](https://developer.apple.com/documentation/ios-ipados-release-notes/ipados-16-release-notes) |
```

Comment:
Looks like this bad date slipped through @Falco20019. We'll have to address that in a distroessed update.